ProcessLib/HeatTransportBHE/HeatTransportBHEProcess.cpp

```diff
@@ -249,9 +249,9 @@ HeatTransportBHEProcess::createBHEBoundaryConditionTopBottom()
     for (unsigned bhe_i = 0; bhe_i < n_BHEs; bhe_i++)
     {
         // get the BHE name
-        auto bhe_name = _process_data._vec_BHE_property.at(bhe_i)->getName();
+        auto bhe_name = _process_data._vec_BHE_property.at(bhe_i)->name;
         // get the BHE type
-        auto bhe_typ = _process_data._vec_BHE_property.at(bhe_i)->getBheType();
+        auto bhe_typ = _process_data._vec_BHE_property.at(bhe_i)->boundary_type;
```

Comment:
`auto const& bhe_boundary_type = _process_data._vec_BHE_property[bhe_i]->boundary_type;`

Reply:
Here bhe_type is needed. Not the boundary type. 
set to auto const& in commit c3f2e3f.